Control coverage/Feature/VirtualMachine.md

```diff
@@ -1456,4 +1457,61 @@ subscription level:
 <br />
 
 ___
+## Azure_VirtualMachine_AuthN_Enable_AAD_Auth_Linux_Trial 
+
+### Display Name 
+[Trial] AAD extension must be deployed to the Linux VM
+
+### Rationale 
+Installing AAD extension on VM allows you to login into VM using Azure AD, making it possible to login user without password and improves authentication security.
+
+### Control Settings 
+```json 
+{
+     "Linux": [
+            "ExtensionType" : "AADSSHLoginForLinux",
+            "ExtensionPublisher" : "Microsoft.Azure.ActiveDirectory",
+            "ProvisioningState" : "Succeeded"
+      ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> AAD extension present and provisioning state is succeeded.
+>
+> **Failed:** 
+>  AAD extension is not present or provisioning state is not succeeded.
+>
+> **NotApplicable:** 
+> Operating System (OS) Windows type is not supported for the evaluation.
+>
+
+### Recommendation
+
+- **Azure Portal** 
+
+	 To install AAD Extension in VM, Go to Azure Portal --> VM --> Settings --> Extensions+Applications --> Click Add --> Select AADSSHForLinuxVM --> Click Next --> Click Review+Create. 
+
+-
+	Install the AAD Auth extension.
+	 
+	```powershell 
+	Set-AzVMExtension -Publisher 'Microsoft.Azure.ActiveDirectory' -Type 'AADSSHLoginForLinux' -Name 'AADSSHLoginForLinux' -TypeHandlerVersion 1.0 -ResourceGroupName 'myResourceGroup' -Location 'myLocation' -VMName 'myVM'
+	 ```
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list Virtual Machine at
+subscription level:
+[/subscriptions/{subscriptionId}/providers/Microsoft.Compute/virtualMachines?api-version=2019-07-01](https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/list-all?tabs=HTTP)<br />
+**Properties:** properties.storageProfile.osDisk.osType
+
+
+- ARM API to list extensions at Virtual Machine level: "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupsName}/providers/Microsoft.Compute/virtualMachines/{VMName}/extensions?api-version=2019-07-01" (https://learn.microsoft.com/en-us/rest/api/compute/virtual-machine-extensions/list?tabs=HTTP)
```

Comment:
Not looking correct after markdown is parsed

Reply:
Fixed and aligned as per other controls.